config/initializers/flipper.rb

```diff
@@ -0,0 +1,6 @@
+class CanAccessFlipperUI
```

Comment:
I think this class can be renamed to `AdminAccessible`, it's more descriptive to what it's actually doing and should be reused if we add any other admin-only route